calendar-bundle/src/EventListener/CalendarFeedListener.php

```diff
@@ -70,7 +73,19 @@ public function onFetchEventsForFeed(FetchEventsForFeedEvent $systemEvent): void
             default => null,
         };
 
-        $calendarEvents = $this->calendarEventsGenerator->getAllEvents($calendars, new \DateTime(), new \DateTime('9999-12-31 23:59:59'), $featured, true, (int) $pageModel->feedRecurrenceLimit);
+        // Create a faux Module instance, so that the getAllEvents hook can be executed
+        $moduleModel = $this->framework->createInstance(ModuleModel::class);
+        $moduleModel->type = 'eventlist';
+        $moduleModel->cal_calendar = $calendars;
+        $moduleModel->cal_noSpan = true;
+        $moduleModel->cal_format = 'next_all';
+        $moduleModel->cal_order = 'ascending';
+        $moduleModel->cal_featured = $pageModel->feedFeatured;
+        $moduleModel->preventSaving(false);
```

Comment:
> This PR fixes that by creating a faux instance to be passed, with the settings that would apply to an event list. Yes, this is hacky - but until we rewrite the event modules to modern fragments etc. I would leave this as is.

@contao/developers Do you agree?

Reply:
If we don't do this, the RSS feed will not contain any dynamically added events via the `getAllEvents` hook. Which is also the case in all previous Contao versions, so if we omit this, nothing will be lost, technically.